examples/containernet_example_cmd_field.py

```diff
@@ -26,6 +26,7 @@
 net.addLink(s2, d2)
 info('*** Starting network\n')
 net.start()
+d2.start()
```

Comment:
I am not 100% sure about the `.start()` as name... While it is consistent with the `net.start()` it may be confusing that we need to start `d2`, but not `d1` here...
So maybe it is worth to break up the consistency here and call it something like `run_docker_cmd`. But this may cause confusion with the existing `cmd()` command which allows execution of arbitrary commands...

So if you have suggestions for a better name, I am glad to hear them :)